test/apps/utils.py

```diff
@@ -152,6 +155,38 @@ def make_schema(endpoints: Tuple[str, ...]) -> Dict:
                     "default": {"description": "Default response"},
                 },
             }
+        elif endpoint == "create_user":
+            schema = {
+                "parameters": [
+                    {
+                        "name": "data",
+                        "in": "body",
+                        "required": True,
+                        "schema": {
+                            "type": "object",
+                            "properties": {"username": {"type": "string", "minLength": 3}},
+                            "required": ["username"],
+                        },
+                    }
+                ],
+                "responses": {
+                    "201": {
+                        "description": "OK",
+                        "x-links": {
```

Comment:
Since there are no links in Open API 2 we need to handle `requestBody` carefully. Either we need to name it `body` and handle like other parameters or still use `requestBody`, however, it is not clear for me how to specify multiple parameters for `requestBody`

Reply:
Noted in the TODO list